R/get_uuid.R

```diff
@@ -43,6 +43,20 @@ get_uuid <- function(name = NULL, n = 1, url = FALSE) {
   # API call -------------------------------------------------------------
   opts <- list()
   if (!is.null(name)) opts$filter_name <- name
+  # Get clade uuid
+  opts$page <- 0
+  opts$embed_items <- "true"
+  api_return <- phy_GET("nodes", opts)
+  clade_uuid <- api_return$`_embedded`$items$uuid
+  if (is.null(clade_uuid)) {
+    stop(paste0("Image resource not available for `name`. \n",
+                "Ensure provided name is a valid taxonomic name or ",
+                "try a species/genus resolution name."))
+  }
```

Comment:
Could we try using the `/autocomplete` endpoint to give some suggestions of what the user might want? I don't think we want to use this all the time to try to guess what the user wants, but when they get no results, it might be nice to give them friendly suggestions as to what to try instead? This should also help emulate the phylopic.org search functionality.